packages/framework/src/Component/Translation/functions.php

```diff
@@ -7,11 +7,11 @@
 /**
  * @param string $id
  * @param array $parameters
- * @param string|null $translationDomain
+ * @param string|null $domain
```

Comment:
Please revert this change, as this naming was intentional to prevent confusion with e-shop domain.

Reply:
However, this is inconsistent with other trans functions used in PHP or backend JS.

Reply:
I have updated phpdoc with description of what you can pass to this parameter. I think, that we should be consistent with parameters naming across these functions.

Reply:
I vote for `$translationDomain` so it is obvious here at least at the most exposed interface (we could rename all the "domain" usages but this one seems crucial. The comment could be a nice solution but it is not visible every time:
![image](https://github.com/user-attachments/assets/47bff8bf-1624-4913-ab59-744d525a5028)

![Screenshot from 2024-12-20 11-22-58](https://github.com/user-attachments/assets/69a52c08-a58b-429d-80e2-7a04d3b906b0)

![image](https://github.com/user-attachments/assets/0ba2c08a-bf1c-423a-8060-07f41bfc0f21)

Reply:
I have asked @vitek-rostislav for 3rd opinion, so we can have decision here and he agreed with me, so please change it as requested.

Reply:
I'm closing this since you backed down after your call with me